src/component/plasmic/flip_card/PlasmicUpdate.css

```diff
@@ -28,31 +28,21 @@
   position: relative;
```

Comment:
The path is this. 

```
src/component/plasmic/flip_card/PlasmicUpdate.css
```

It would be cool to not have dashes or underscores in package names if possible and have flat hierarchies so that we only have the index files. 

```
src/component/plasmic/flipcard/index.css
```